DOC-3300: Updated title in the getting started page, as well as the description and navbar. (#3876)
tinymce/tinymce-docs: modules/ROOT/partials/what-is-tinymce.adoc

> An introduction to {productname}, the rich-text editor that allows users to create formatted content within a user-friendly interface.

Reviewer: This change probably isn't necessary

Reviewer 2: I agree with @MitchC1999 on this, also if we are changing this for SEO reasons, please create followup tickets for `5 | 6 | 7` branches also.

Author: @MitchC1999 @kemister85 The reason for that change was due to a request from @mikegauci-tiny from the DX team.

Reviewer: I get changing the heading but why the subheading? This wouldn't make sense on the page cos it's not introducing anything.

Author: I'll ask @mikegauci-tiny about this and get back to you.

Reviewer: I already asked, don't worry. If there's a reason then I'll resolve this conversation

Reviewer: @abhinavgandham could you revert this change?

Reviewer 2: ```suggestion
{productname} is a rich-text editor that allows users to create formatted content within a user-friendly interface.
```
